test/run_test.py

```diff
@@ -1314,6 +1324,12 @@ def main():
         # downloading test cases configuration to local environment
         get_test_case_configs(dirpath=test_directory)
 
+    if options.dynamo:
+        torch.testing._internal.common_utils.TEST_WITH_TORCHDYNAMO = True
+
+    if options.inductor:
```

Comment:
Nit:  Does these two options mutually exclusive (only one can be set)?  If yes, you could add https://docs.python.org/3/library/argparse.html#mutual-exclusion to prevent people accidentally setting them both